BUILD.bazel

```diff
@@ -275,8 +280,71 @@ cc_binary(
     data = [":p4include"],
 )
 
-# This builds the p4test backend.
+# dpdk backend
```

Comment:
perhaps for the bazel build you want to request a review from @smolkaj

Reply:
I am unable to add @smolkaj to reviewer list. may be due to permission issues. Can you please add him?

Reply:
That mention is enough, he had been notified. If he doesn't reply in a couple of days I will merge it anyway

Reply:
Got it, taking a look :)